src/vm/stackwalk.h

```diff
@@ -45,6 +45,10 @@ class AppDomain;
  #endif 
 #endif // _TARGET_X86_ && !FEATURE_PAL
 
+#if !defined(_TARGET_X86_)
+#define RECORD_RESUMABLE_FRAME_SP
```

Comment:
@parjong have you tried if enabling it for x86 Linux too would work? It is used just as a safeguard and it seems to me that the reason why it was excluded for x86 is the Windows x86 EH and that it can be applied to x86 Linux.